### PR TITLE
add table style plan on tidb dashboard

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
@Oreoxmt @baurine PTAL

File: dashboard/dashboard-slow-query.md [hunk 1]
```diff
@@ -64,11 +64,29 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 
 ### 执行计划
 
-TiDB Dashboard 提供两种方式查看执行计划：图形和文本。通过图形化的执行计划，你可以更清晰地了解一个语句的具体算子和对应的内容。参阅[理解 TiDB 执行计划](/explain-overview.md)。
+执行计划可以被三种形态查看：表格、文本和图形。参阅[理解 TiDB 执行计划](/explain-overview.md)文档了解如何解读执行计划。
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:67]:
```suggestion
TiDB Dashboard 提供三种方式查看执行计划：表格、文本和图形。关于如何解读执行计划，请参考[理解 TiDB 执行计划](/explain-overview.md)。
```

File: dashboard/dashboard-slow-query.md [hunk 2]
```diff
@@ -64,11 +64,29 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
 
-#### 图形化执行计划介绍
+#### 表格形态的执行计划介绍
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:69]:
```suggestion
#### 表格形态的执行计划
```

File: dashboard/dashboard-slow-query.md [hunk 3]
```diff
@@ -64,11 +64,29 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+- 列宽可以自由拖拽。
+- 自动截断超过列宽的内容，并提供悬浮窗展示完整信息。
+- 如果执行计划过长，通过 txt 格式下载到本地分析。
+- 支持隐藏任意列，并且能够在列选择中进行管理。
+
+![可视化执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)
+
+
+#### 图形形态的执行计划介绍
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:86]:
```suggestion
#### 图形形态的执行计划
```

File: dashboard/dashboard-slow-query.md [hunk 4]
```diff
@@ -64,11 +64,29 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+下图为一个表格形态的执行计划展示：
+
+![可视化执行计划](/media/dashboard/dashboard-table-plan.png)
+
+表格形态的展示内容与文本形态相似，但是有更易用的交互：
+
+- 列宽可以自由拖拽。
+- 自动截断超过列宽的内容，并提供悬浮窗展示完整信息。
+- 如果执行计划过长，通过 txt 格式下载到本地分析。
+- 支持隐藏任意列，并且能够在列选择中进行管理。
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:81]:
```suggestion
表格形态的执行计划提供详细的执行计划信息，便于你快速识别算子指标的异常情况，以及对比不同算子的状态。下图是一个表格形态的执行计划示例：

![表格形态的执行计划](/media/dashboard/dashboard-table-plan.png)

表格形态展示的内容与文本形态类似，但具有更易用的交互：

- 支持自由调整列宽
- 当内容超出列宽时，支持自动截断并提供悬浮窗展示完整信息
- 如果执行计划内容较多，可以下载 txt 格式到本地分析
- 允许隐藏任意列，并且能够在列选择中进行管理
```

File: dashboard/dashboard-slow-query.md [hunk 5]
```diff
@@ -64,11 +64,29 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+![可视化执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)
+
+
+#### 图形形态的执行计划介绍
+
+图形形态的执行计划，更适合于了解一个复杂 SQL 的执行计划树宏观的样貌，并且从宏观到微观的了解每一个的算子和对应的内容。
+下图为一个图形形态的执行计划展示：
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:89]:
```suggestion
图形形态的执行计划更适合宏观地查看一个复杂 SQL 的执行计划树，并且详细了解每个算子及对应的内容。下图是一个图形形态的执行计划示例：
```

File: dashboard/dashboard-statement-details.md [hunk 6]
```diff
@@ -69,13 +69,60 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+![可视化执行计划弹窗](/media/dashboard/dashboard-visual-plan-2.png)
+
+- 执行计划的展示顺序是从左到右，从上到下。
+- 上面的节点是父算子，下面的节点是子算子。
+- 节点顶栏的颜色代表算子执行的组件：黄色代表 TiDB，蓝色代表 TiKV，粉色代表 TiFlash。
+- 节点的顶栏为算子的名称，正文为算子的基本信息。
+
+点击节点区域，右侧将弹出算子的详细信息。
+
+![可视化执行计划弹窗-侧栏](/media/dashboard/dashboard-visual-plan-popup.png)
```
Comment (Oreoxmt) [inline dashboard/dashboard-statement-details.md:111]:
```suggestion
TiDB Dashboard 提供三种方式查看执行计划：表格、文本和图形。关于如何解读执行计划，请参考[理解 TiDB 执行计划](/explain-overview.md)。

#### 表格形态的执行计划

表格形态的执行计划提供详细的执行计划信息，便于你快速识别算子指标的异常情况，以及对比不同算子的状态。下图是一个表格形态的执行计划示例：

![表格形态的执行计划](/media/dashboard/dashboard-table-plan.png)

表格形态展示的内容与文本形态类似，但具有更易用的交互：

- 支持自由调整列宽
- 当内容超出列宽时，支持自动截断并提供悬浮窗展示完整信息
- 如果执行计划内容较多，可以下载 txt 格式到本地分析
- 允许隐藏任意列，并且能够在列选择中进行管理

![表格形态的执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)

#### 图形形态的执行计划

图形形态的执行计划更适合宏观地查看一个复杂 SQL 的执行计划树，并且详细了解每个算子及对应的内容。下图是一个图形形态的执行计划示例：

![图形形态的执行计划](/media/dashboard/dashboard-visual-plan-2.png)

- 执行计划的展示顺序是从左到右，从上到下。
- 上面的节点是父算子，下面的节点是子算子。
- 节点顶栏的颜色代表算子执行的组件：黄色代表 TiDB，蓝色代表 TiKV，粉色代表 TiFlash。
- 节点的顶栏为算子的名称，正文为算子的基本信息。

点击节点区域，右侧将弹出算子的详细信息。

![图形形态的执行计划-侧栏](/media/dashboard/dashboard-visual-plan-popup.png)
```

File: dashboard/dashboard-slow-query.md [hunk 7]
```diff
@@ -64,11 +64,27 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+表格形态展示的内容与文本形态类似，但具有更易用的交互：
+
+- 支持自由调整列宽
+- 当内容超出列宽时，支持自动截断并提供悬浮窗展示完整信息
+- 如果执行计划内容较多，可以下载 txt 格式到本地分析
+- 允许隐藏任意列，并且能够在列选择中进行管理
+
+![可视化执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)
+
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:83]:
```suggestion
![表格形态的执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)
```

File: dashboard/dashboard-slow-query.md [hunk 8]
```diff
@@ -64,11 +64,27 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:84]:
```suggestion
```

File: dashboard/dashboard-slow-query.md [hunk 9]
```diff
@@ -64,11 +64,25 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+![表格形态的执行计划](/media/dashboard/dashboard-table-plan.png)
+
+表格形态展示的内容与文本形态类似，但具有更易用的交互：
+
+- 支持自由调整列宽
+- 当内容超出列宽时，支持自动截断并提供悬浮窗展示完整信息
+- 如果执行计划内容较多，可以下载 txt 格式到本地分析
+- 允许隐藏任意列，并且能够在列选择中进行管理
+
+![表格形态的执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:82]:
```suggestion
![表格形态的执行计划-选择列](/media/dashboard/dashboard-table-plan-columnpicker.png)

```

**Review by Oreoxmt — APPROVED** [review]

File: media/dashboard-slow-queries-detail-plans-basic.png
Comment (Oreoxmt) [inline media/dashboard-slow-queries-detail-plans-basic.png:1]: @lilyjazz Please move these images to the `media/dashboard` folder.

**Review by baurine — APPROVED** [review]

File: dashboard/dashboard-statement-details.md [hunk 10]
```diff
@@ -69,13 +69,57 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
```
Comment (Oreoxmt) [inline dashboard/dashboard-statement-details.md:69]: @lilyjazz Please update L67:

```
- 执行计划：执行计划的完整内容，有图形和文本两种展示形式。参阅[理解 TiDB 执行计划](/explain-overview.md)文档了解如何解读执行计划。如果选择了多个执行计划，则显示的是其中任意一个。
```

File: media/dashboard-slow-queries-detail-plans-basic.png
Comment (lilyjazz) [reply media/dashboard-slow-queries-detail-plans-basic.png:1]: OK

File: dashboard/dashboard-slow-query.md [hunk 11]
```diff
@@ -83,6 +98,34 @@ TiDB Dashboard 提供两种方式查看执行计划：图形和文本。通过
...
-点击标签页标题可切换显示不同分类的 SQL 执行信息：
+其他关于该 SQL 的基本信息、执行时间、Coprocessor 读取、事务、报错等信息，可点击相应标签页标题切换。
 
 ![显示不同分类执行信息](/media/dashboard/dashboard-slow-queries-detail2-v620.png)
+
+#### 基本信息
+
+包含关于表名、索引名、执行次数、累计耗时等信息。**描述** (Description) 列对各个字段进行了具体描述。
+
+![基本信息](/media/dashboard/dashboard-slow-queries-detail-plans-basic.png)
```
[the remark this answers is not in the document]
Comment (lilyjazz) [reply dashboard/dashboard-slow-query.md:109]: 挪到 dashboard 文件夹里面吧

**Review by ran-huang — APPROVED** [review]

**Pull request merged**